Packages/com.unity.inputsystem/InputSystem/InputUpdateType.cs

```diff
@@ -71,6 +71,8 @@ public enum InputUpdateType
         /// </remarks>
         ManualThreaded = Manual | 1 << 5,
 
+        RunInBackground = 1 << 31,
```

Comment:
I need to add this back in every time I want to debug, because C#'s debugging and watch window info only shows known enumeration values.
So: Default | (1 << 31)
will display in the watch window as 'Default', which makes debugging and reading the flags a real pain.